### PR TITLE
fix:projects folder structure 

### PR DESCRIPTION
**Pull request by oseijoel6111**
made changes to previous projects folder structure

### CODE REVIEW

File: src/api/routes/index.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { Router, Request, Response } from "express";
 import { PORT } from "../../utils/env";
 import { CHECKAPPTOKEN } from "../middlewares/CheckApp";
 
-import User from "./userRoute";
+import User from "./UserRoute";
 import Book from "./bookRoute";
```
Comment (Mancunia) [inline src/api/routes/index.ts:6]: If you are going for PascalCase, then let it run across, I went for camelCase, so let's stick to one

File: src/index.ts [hunk 2]
```diff
@@ -16,7 +16,8 @@ Mongoose.connection
     console.log("Connected to MongoDB");
     app.emit("ready");
   })
-  .on("error", () => {
+  .on("error", (e) => {
+    console.log(e);
     console.log("Couldn't connect to DB");
```
Comment (Mancunia) [inline src/index.ts:21]: Cool, but let's take off the comments afterwards don't do the same mistakes I did 😊

**Review by Mancunia — COMMENTED** [review]
Boss please check the comments and let's go

File: src/api/routes/index.ts (hunk 1 above)
Comment (oseijoel6111) [reply src/api/routes/index.ts:6]: Ohh okay, that's true, then I will resolve that. Thank you.

**Pull request merged**